src/common/models/Cookie.class.js

```diff
@@ -398,6 +398,8 @@ class Cookie {
     this.session.gar.responsiblePerson.responsibleCounty = person.responsibleCounty;
     this.session.gar.responsiblePerson.responsibleContactNo = person.responsibleContactNo;
```

Comment:
In `initialiseGar`, fixedBasedOperator and fixedBasedOperatorAnswer are not part of `this.session.gar.responsiblePerson.` e.g.
```
initialiseGar() {
    if (this.session.gar == null) {
      this.session.gar = {{
...
        responsiblePerson: {
          responsibleGivenName: null,
          responsibleSurname: null,
          responsibleAddressLine1: null,
          responsibleAddressLine2: null,
          responsibleTown: null,
          responsibleCounty: null,
          responsiblePostcode: null,
          responsibleContactNo: null,
          responsibleEmail: null,
        },
        fixedBasedOperator: null,
        fixedBasedOperatorAnswer: null,
        fixedBasedOperatorList: [],
        ...
      };
    }
}
```
Should this be changed?
* It is unlikely to cause a problem now, but should be changed as initially `this.session.gar.responsiblePerson.fixedBasedOperatorAnswer` it will be `undefined` instead of `null`, potentially causing bugs in the future.
* If the above is incorrect logic, let me know.

Reply:
You're right.  Well spotted.